src/components/Header/Header.jsx

```diff
@@ -1,5 +1,22 @@
+import mainLogo from '../../assets/img/logo.png';
+import commonStyles from '../../styles.scss';
+import classes from './Header.module.scss';
+
+import LocationSearch from './components/LocationSearch/LocationSearch';
+import MainNavigation from './components/MainNavigation/MainNavigation';
+import CurrentTime from '../CurrentTime/CurrentTime';
+
 function Header() {
-  return <div>Header</div>;
+  return (
+    <header className={classes.header}>
+      <a className={classes.logo} href="/">
+        <img src={mainLogo} alt="" />
```

Comment:
always assign alt

Reply:
In ideal World, you'll need to add all accessibility attributes